### PR TITLE
add backup restore document

### PR DESCRIPTION
**Pull request by sunhao2017**
add backup restore document

### CODE REVIEW

File: op-guide/backup-restore.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+
+在这个备份恢复过程中，我们会用到下面的工具:
+
+- mydumper 从 MySQL 导出数据
+- loader 导入数据到 TiDB
+- pump 用户生成binlog文件
+- drainer 用于增量导入到新 TIDB
+
+### 两种迁移场景
```
Comment (queenypingcap) [inline op-guide/backup-restore.md]: This document is about Backup and Restore, not migration.

File: op-guide/backup-restore.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+title: 备份恢复
+category: advanced
+---
+
+# 备份恢复
+
+## 概述
+
+
+该文档详细介绍了如何对 TiDB 进行备份恢复。本文档暂时只考虑全量备份与恢复。
```
Comment (queenypingcap) [inline op-guide/backup-restore.md]: This is contradictory with what's documented in the following part...

**Comment by queenypingcap** [comment]
@shenli @siddontang PTAL.

File: op-guide/backup-restore.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+这里我们假定 TiDB 服务信息如下：
+
+|Name|Address|Port|User|Password|
+|----|-------|----|----|--------|
+|TiDB|127.0.0.1|4000|root|*|
+
+
+在这个备份恢复过程中，我们会用到下面的工具:
+
+- mydumper 从 MySQL 导出数据
```
Comment (queenypingcap) [inline op-guide/backup-restore.md]: Why exporting data from MySQL?

File: op-guide/backup-restore.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+sha256sum -c tidb-enterprise-tools-latest-linux-amd64.sha256
+# 解开压缩包
+tar -xzf tidb-enterprise-tools-latest-linux-amd64.tar.gz
+cd tidb-enterprise-tools-latest-linux-amd64
+```
+
+
+## 使用 `mydumper`/`loader` 全量导入数据
+
+`mydumper` 是一个强大的数据迁移工具，具体可以参考 [https://github.com/maxbube/mydumper](https://github.com/maxbube/mydumper)。
```
Comment (queenypingcap) [inline op-guide/backup-restore.md]: Remove the statement about "迁移"

File: op-guide/backup-restore.md (hunk 2 above)
Comment (sunhao2017) [inline op-guide/backup-restore.md]: already remove the Incremental backup part

File: op-guide/backup-restore.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
+---
+title: 备份恢复
```
Comment (queenypingcap) [inline op-guide/backup-restore.md]: -> 备份与恢复

File: op-guide/backup-restore.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+这里我们假定 TiDB 服务信息如下：
+
+|Name|Address|Port|User|Password|
+|----|-------|----|----|--------|
+|TiDB|127.0.0.1|4000|root|*|
+
+
+在这个备份恢复过程中，我们会用到下面的工具:
+
+- mydumper 从 TiDB 导出数据
```
Comment (queenypingcap) [inline op-guide/backup-restore.md]: Please change this statement

File: op-guide/backup-restore.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+ - mydumper 导出后总数据量 214G，单表 8 列，20 亿行数据
+ - 集群拓扑
+     - TIKV * 12
+     - TIDB * 4 
+     - PD * 3
+ - mydumper -F 设置为 16, loader -t 参数 64
+ 
+结果：导入时间 11 小时左右，19.4 G/小时
+
+### 从 TiDB 备份数据
```
Comment (queenypingcap) [inline op-guide/backup-restore.md]: Is it possible to support automatic backup?

File: op-guide/backup-restore.md (hunk 2 above)
Comment (queenypingcap) [inline op-guide/backup-restore.md]: Looks like MySQL has incremental backup and restore...
https://dev.mysql.com/doc/refman/5.7/en/point-in-time-recovery.html

File: op-guide/backup-restore.md (hunk 7 above)
Comment (sunhao2017) [inline op-guide/backup-restore.md]: It's not the concern of this document.

**Comment by sunhao2017** [comment]
All your suggestions have been modified
@shenli @queenypingcap PTAL.

File: op-guide/backup-restore.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+我们使用 `mydumper` 从 TiDB 导出数据进行备份，然后用 `loader` 将其导入到 TiDB 里面进行恢复。
+
+> 注意：虽然 TiDB 也支持使用 MySQL 官方的 `mysqldump` 工具来进行数据的备份恢复工作，但相比于 `mydumper` / `loader`，性能会慢很多，大量数据的备份恢复会花费很多时间，这里我们并不推荐。
+
+### `mydumper`/`loader` 全量备份恢复最佳实践
+为了快速的备份恢复数据 (特别是数据量巨大的库), 可以参考下面建议
+
+* 使用 mydumper 导出来的数据文件尽可能的小, 最好不要超过 64M, 可以设置参数 -F 64 
+* loader的 -t 参数可以根据 tikv 的实例个数以及负载进行评估调整，例如 3个 tikv 的场景， 此值可以设为 3 *（1 ～ n)；当 tikv 负载过高，loader 以及 tidb 日志中出现大量 `backoffer.maxSleep 15000ms is exceeded` 可以适当调小该值，当 tikv 负载不是太高的时候，可以适当调大该值。
```
Comment (andelf) [inline op-guide/backup-restore.md]: quote arithmetic expression in 

    ``

use ascii `()` instead of `（）`

**Comment by shenli** [comment]
@sunhao2017 Please resolve the conflicts.

**Pull request merged**